directory-listing/darkmode.css

```diff
@@ -0,0 +1,28 @@
+body{
+  background-color: #333333;
+  border-top: 0.5rem solid #242424;
+  border-bottom: 0.5rem solid #242424;
+}
+
+table tr:hover {
+  background-color: #434343 !important;
+}
+
+a, h1, td, address{
+  color: #e2e2e2 !important;
```

Comment:
Page wide foreground and background colors should be set for `body`. And you don't need `!important`.

Reply:
You are correct for all the elements beside the `a` tag - since the 'darkmode' is an add-on to the original layout, I have to override the color (`!important` not needed)